assembly/src/parsers/nodes.rs

```diff
@@ -503,6 +517,19 @@ impl fmt::Display for Instruction {
     }
 }
 
+// HELPER FUNCTIONS
+// ================================================================================================
+
+/// Builds a string from input vector to display push operation
+fn display_push_vec<T: fmt::Display>(f: &mut fmt::Formatter<'_>, values: &Vec<T>) -> fmt::Result {
```

Comment:
If you change `&Vec<T>` to `&[T]`, you should be able to use this function for `PushWord` as well.